docs/learn/quotas.rst

```diff
@@ -83,6 +83,16 @@ error message under the **Filter errors by error message** section.
 
 .. note:: Filter by error message is avaliable only on Large and Enterprise Plans
 
+Filter by issue
+~~~~~~~~~~~~~~~
+
+If there is an issue that you unable to take action on but that continues to occur, you
+can delete and discard it. This will delete most data associated with an issue and
```

Comment:
Can we add a short instruction here, ie 'you can delete and discard it by clicking 'Delete and Discard' on the issue details page.'?